go/client/types_test.go

```diff
@@ -8,60 +8,11 @@ import (
-		"b6ky9ApYa/ZFecNmkzld8tilDsKH7GAdr2vUz0W8bR6YlY0cgNQ/KXrFKL5giaqUt9w5OThr" +
-		"WaGEDNufSDin4AChHzPhncfjwD3DZFfjcDrR9H7xryZSWVZUYTLo7Vs/ceuWvAkuh2yZe1QS" +
-		"d5XKKy52MqAwqYG4Ioi2cQfCgVEe2P8HEj1XzlYxHOD0ohNf6IRnPrGVHSTcllyeJP5uvU/e" +
-		"6CiOUe0F+f98I02F18cDbDfleRc6u03idR3q4ZL9AgMBAAGjggGyMIIBrjAPBgNVHRMBAf8E" +
-		"BTADAQEAMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjAOBgNVHQ8BAf8EBAMCBaAw" +
-		"NgYDVR0fBC8wLTAroCmgJ4YlaHR0cDovL2NybC5nb2RhZGR5LmNvbS9nZGlnMnMxLTI3LmNy" +
-		"bDBTBgNVHSAETDBKMEgGC2CGSAGG/W0BBxcBMDkwNwYIKwYBBQUHAgEWK2h0dHA6Ly9jZXJ0" +
-		"aWZpY2F0ZXMuZ29kYWRkeS5jb20vcmVwb3NpdG9yeS8wdgYIKwYBBQUHAQEEajBoMCQGCCsG" +
-		"AQUFBzABhhhodHRwOi8vb2NzcC5nb2RhZGR5LmNvbS8wQAYIKwYBBQUHMAKGNGh0dHA6Ly9j" +
-		"ZXJ0aWZpY2F0ZXMuZ29kYWRkeS5jb20vcmVwb3NpdG9yeS9nZGlnMi5jcnQwHwYDVR0jBBgw" +
-		"FoAUQMK9J47MNIMwojPX+2yz8LQsgM4wJwYDVR0RBCAwHoILdHJpc3VyZS5jb22CD3d3dy50" +
-		"cmlzdXJlLmNvbTAdBgNVHQ4EFgQUncM2IqTKpShSWBCF37hH/dDVbR8wDQYJKoZIhvcNAQEL" +
-		"BQADggEBAEiIEjD9CWyDyV27csg6itq48yOF/icQ6j3Y8rmyQ1levCDGaR7tv4RjU/iuQEwR" +
-		"hGOG3xQ7So+qSKm0lj6ZJJpv3nLroQKpcadyW6n/s4CokHOgxxlzhwvdeTXvul0kt3QG8l4s" +
-		"HgzGqvfnjUsqljQ5U4Z2BAsRuAiilVc0/TTPbb0smbnq4GFbOCXe73xFgY4NZJ6IPzvwhTdT" +
-		"Lxg0dUi5yOgjsrJ7agV1sI+Wk1C7+Y70FOHcM3vNC4HZ2KIbM/pex8IH64J9/TYEkVCtTvdB" +
-		"tXjj+06auSnL+GOFJgqzSfbTWT8AJIbo0GezGpN88hwulluXnhh2vPgqSvjSBjIAAA=="
+	PrecertEntryB64          = "AAAAAAFLSYHwyAABN2DieQ8zpJj5tsFJ/s/KOZOVS1NvvzatRdCoQVt5M30ABHowggR2oAMCAQICEAUyKYw5aj4l/KoZd+gntfMwDQYJKoZIhvcNAQELBQAwbTELMAkGA1UEBhMCVVMxFjAUBgNVBAoTDUdlb1RydXN0IEluYy4xHzAdBgNVBAsTFkZPUiBURVNUIFBVUlBPU0VTIE9OTFkxJTAjBgNVBAMTHEdlb1RydXN0IEVWIFNTTCBURVNUIENBIC0gRzQwHhcNMTUwMjAyMDAwMDAwWhcNMTYwMjI3MjM1OTU5WjCBwzETMBEGCysGAQQBgjc8AgEDEwJHQjEbMBkGCysGAQQBgjc8AgECFApDYWxpZm9ybmlhMR4wHAYLKwYBBAGCNzwCAQEMDU1vdW50YWluIFZpZXcxCzAJBgNVBAYTAkdCMRMwEQYDVQQIDApDYWxpZm9ybmlhMRYwFAYDVQQHDA1Nb3VudGFpbiBWaWV3MR0wGwYDVQQKDBRTeW1hbnRlYyBDb3Jwb3JhdGlvbjEWMBQGA1UEAwwNc2RmZWRzZi50cnVzdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALGdl97zn/gpxl6gmaMlcpizP/Z1RR/cVkGiIjR67kpWIB9MGkBvLxmBXYbewaYRdo59VWyOM6fxtMeNsZzOrlQOl64fBmCy7k+M/yBFuEqdoig0l0RAbs6u0LCNRv2rNUOz2G6nCGJ6YaUpt5Onatxrd2vI1bPU/iHixKqSz9M7RedBIGjgaDor7/rR3y/DILjdvwL/tgPSz3R5gnf9lla1rNRWWbDl12HgLc+VxTCVVVqTGtW/qbSWfARdXxLeLWtTfNk68q2LReVUC9QyeYdtE+N2+2SXeOEN+lYWW5Ab036d7k5GAntMBzLKftZEkYYquvaiSkqu2PSaCSLKT7UCAwEAAaOCAdEwggHNMEcGA1UdEQRAMD6CDWtqYXNkaGYudHJ1c3SCC3NzZGZzLnRydXN0gg1zZGZlZHNmLnRydXN0ghF3d3cuc2RmZWRzZi50cnVzdDAJBgNVHRMEAjAAMA4GA1UdDwEB/wQEAwIFoDArBgNVHR8EJDAiMCCgHqAchhpodHRwOi8vZ20uc3ltY2IuY29tL2dtLmNybDCBoAYDVR0gBIGYMIGVMIGSBgkrBgEEAfAiAQYwgYQwPwYIKwYBBQUHAgEWM2h0dHBzOi8vd3d3Lmdlb3RydXN0LmNvbS9yZXNvdXJjZXMvcmVwb3NpdG9yeS9sZWdhbDBBBggrBgEFBQcCAjA1DDNodHRwczovL3d3dy5nZW90cnVzdC5jb20vcmVzb3VyY2VzL3JlcG9zaXRvcnkvbGVnYWwwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMB8GA1UdIwQYMBaAFLFplGGr5ssMTOdZr1pJixgzweFHMFcGCCsGAQUFBwEBBEswSTAfBggrBgEFBQcwAYYTaHR0cDovL2dtLnN5bWNkLmNvbTAmBggrBgEFBQcwAoYaaHR0cDovL2dtLnN5bWNiLmNvbS9nbS5jcnQAAA=="
```

Comment:
gofmt is fine with them.
Go source [explicitly] has no line length limit, and I figured that like this it'll only take 1 line in most editors, rather than a wall of base64 to get past.